utils/tests/artifactoryconsts.go

```diff
@@ -132,6 +132,7 @@ const (
 	WinSimpleUploadSpec                    = "win_simple_upload_spec.json"
```

Comment:
I guess we better rename this file from "artifactoryconsts.go " to "consts.go", because it also includes Xray consts.